### ISSUE TITLE
Containers ID in HELO headers

### ISSUE THREAD

**Issue by nemurasu** [getsentry/self-hosted#2182]
### Self-Hosted Version

22.3.0

### CPU Architecture

x86_64

### Docker Version

19.03.13, build 4484c46d9d

### Docker Compose Version

1.29.2

### Steps to Reproduce

1.  Install Sentry using Docker
2.  Install Sentry using Docker
3. Configure it to use some SMTP server
4. Try to send test email
5. Snoop on the communication between Sentry and SMTP server

   
    
    
    

### Expected Result

It should be possible to specify domain used in ELHO command in configuration if it can't be autodetected.

### Actual Result

The question is exactly the same as this one: https://github.com/getsentry/sentry/issues/16386
Эти опции прстутсвовали в конфиге sentry/sentry.conf.example.py:
SENTRY_OPTIONS["mail.list-namespace"] = env('SENTRY_MAIL_HOST', 'localhost')
SENTRY_OPTIONS["mail.from"] = f"sentry@{SENTRY_OPTIONS['mail.list-namespace']}"

Also i tried to directly specify the mail address in these directives:
./sentry/config.yml:mail.from: 'sentry@MYDOMAIN'
./sentry/sentry.conf.py:SENTRY_OPTIONS["mail.from"] = f"sentry@MYDOMAIN"
./sentry/sentry.conf.example.py:SENTRY_OPTIONS["mail.from"] = f"sentry@MYDOMAIN"
./sentry/config.yml:mail.from: 'sentry@MYDOMAIN'
./docker-compose.yml:    hostname: 'MYDOMAIN'
./.env:SENTRY_MAIL_HOST=MYDOMAINThe question is exactly the same as this one.

But all the same, the id of the web and exim4 containers are indicated in the headers of letters from the sentry:
Received: from sentry-self-hosted_worker_1.sentry-self-hosted_default ([172.19.0.28] helo=6ca37a6a566e)
	by e1ec35835910 with esmtp (Exim 4.92)

# docker ps | grep -E "e1ec35835910|6ca37a6a566e"
6ca37a6a566e        getsentry/sentry:22.3.0                  "/etc/sentry/entrypo…"   2 hours ago         Up 27 minutes             9000/tcp                       sentry-self-hosted_worker_1
e1ec35835910        tianon/exim4                             "docker-entrypoint.s…"   2 hours ago         Up 27 minutes             25/tcp                         sentry-self-hosted_smtp_1

Where do I need to specify the domain so that it is in the HELO headers?

**Comment by hubertdeng123** [comment]
Are you using an external SMTP server? Then, you'll need to ignore the built in smtp server (comment it out in your docker-compose.yml file)

https://develop.sentry.dev/self-hosted/email/

**Comment by nemurasu** [comment]
No, initially I wanted to use the internal sentry smtp server. So the question is how, with this use, to configure the sentry so that it points to the HELO FQDN from the SENTRY_MAIL_HOST directive in .env, and not the container ID

**Comment by hubertdeng123** [comment]
Seeing that you've opened this issue here:
https://github.com/getsentry/self-hosted/issues/2189

Are you still experiencing this issue?

**Comment by nemurasu** [comment]
> Seeing that you've opened this issue here: #2189
> 
> Are you still experiencing this issue?

These are problems from different sentry installations and they are not related to each other in any way.

**Comment by aldy505** [comment]
Hopefully #3746 solved this issue. I'm closing this issue until further complaints.
